src/actions/filters.js

```diff
@@ -0,0 +1,9 @@
+export const setTextFilter = (text = '') => ({
```

Comment:
I'm not sure what this is doing for us.

Reply:
@Br3nda This was going to be an action but I've decided to remove it.